Elle624/ivolunteer
# Fix/reload postings view bug

#### What's this PR do?
-Disables submit button if posting has already been added to the users profile

#### What does it fix?
-posting view bug on reload

#### Where should the reviewer start?
-line 5 `postingView.js`
-line 26 `App.js`

#### How should this be manually tested?
-Test that bugs are no longer apparent. When in postingView.js reload the page and it should load with no wrrors.
-Test that you cannot submit once a user already has a posting from an event in upcoming jobs
